app/models/schema/common.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+##
+# Defining a common superset of attributes for Resources
+module Schema
+  module Common
+    extend ActiveSupport::Concern
+
+    ##
+    # Retrieve the attributes from Hyrax, imported from external systems, and local
+    def self.attributes
+      Plum.attributes + Schema::IIIF::Book.attributes
+    end
+
+    included do
+      Common.attributes.each do |field|
+        attribute field
```

Comment:
Apologies, it is likely to continue to block this work, as I would likely need to abandon any (simple) approach using Modules.  Within the scope of `included`, I do not believe that I can easily reference the Module being mixed into the Class.  From what I could determine, this is not a limitation of `ActiveSupport::Concern`:

https://gist.github.com/jrgriffiniii/8cf9ab64d4ecaff44a2f59284d23ba85